tests/rpc/test_rpc_telegram.py

```diff
@@ -188,8 +188,8 @@ def test_status(default_conf, update, mocker, fee, ticker,) -> None:
     telegram = Telegram(freqtradebot)
 
     # Create some test data
-    for _ in range(3):
-        freqtradebot.process_maybe_execute_buys()
+    n = freqtradebot.enter_positions()
```

Comment:
I'm not sure about this change (was it really redundant to call process_maybe_execute_buys() here multiple times?)
Pls review this separately...